gulp.config.js

```diff
@@ -81,12 +42,109 @@ module.exports = function() {
             // Seems to break sourcemaps, so conditionally minify css in gulpfile
             // outputStyle: 'compressed',
             precision: 8,
+            includePaths: [src + theme, src + base]
         },
         uglifySettings: {
             mangle: false,
         },
     };
 
+    setConfigPaths();
     return config;
+
+    //////////////////////////////////
+
+    // Fill the config with themes paths
+    function setConfigPaths() {
+        addThemePaths('./' + src + config.base + '/');
+        addThemePaths('./' + src + config.theme + '/');
+
+        // Styles root
+        var themeStylesRoot = './' + src + config.theme + '/app.scss';
+        var match = glob.sync(themeStylesRoot);
+        config.styles.root = (match.length) ? themeStylesRoot : './' + src + config.base + '/app.scss';
+
+        return config;
+    }
+
+    // Adds paths of a single theme to the config
+    function addThemePaths(themeDir) {
+        // Theme paths patterns
+        var themePaths = {
+            fonts: themeDir + '_fonts/**/*.{otf,svg,eot,ttf,woff,woff2}',
+            html: {
+                root: themeDir + '*.html',
+                nonRoot: [
+                    themeDir + '**/*.html',
+                    '!' + themeDir + email + '**/*',
+                    '!' + themeDir + '*.html',
+                ],
+            },
+            media: [
+                // Ignore media, declaring the _images folder directly moves all of its contents
+                // without the container folder getting in the way
```

Comment:
I'm not sure I fully understand the comment.  Is the use of a `_images` folder a backdoor for theme creators to have their media moved to **the right place** in Ottemo by gulp?